Test/shared/ercFunctions.js

```diff
@@ -25,7 +25,7 @@ async function increaseAllowanceTest(ERC20TokenContract, account, spenderAddress
         spenderAddress,
         amountToIncrease
     );
-    await checkTxnResult(inputIncreaseAllowance, account, ethers, ethers.provider, errMsg);
+    await testHelper.checkTxnResult(inputIncreaseAllowance, account, ethers, ethers.provider, errMsg);
```

Comment:
try not to pass ethers, ethers.provider like this. The helper can use either directly from ethers.js lib or got initialized  itself.

Doing this will cause a lot of problem when we want to use fo other networks